cluster/gce/gci/configure-helper.sh

```diff
@@ -1214,6 +1298,10 @@ function create-master-etcd-apiserver-auth {
    fi
 }
 
+function create-master-network-proxy-apiserver-auth {
+  echo TODO: implement create-master-network-proxy-apiserver-auth
```

Comment:
Does this PR function without this being implemented?

Reply:
If so, make this an issue instead of leaving TODOs in logs.